src/software/test_util/test_util.h

```diff
@@ -13,239 +13,233 @@
 #include "software/world/team.h"
 #include "software/world/world.h"
 
-namespace Test
+/**
+ * This util class is to provide utility functions to our unit test suite,
+ * primarily for assisting with test setup (such as creating World or Field objects)
+ */
+class TestUtil
```

Comment:
Just to clarify, @jonathanlew @akhilveeraghanta you guys have chosen to use static classes over namespaces so we have no namespaces at all?

Reply:
yeah that's a fair point, we should make TestUtil a namespace so we can put equalWithinTolerance in a different file. @akhilveeraghanta thoughts?

Reply:
This is ultimately your guys' decision, but I'd prefer to use namespaces here because I think its communicates the intent better (ie. these are free functions, they have no connection to a "class" or "interface", or any potential class members).

Your point about splitting files is also a great point.